openedx/features/course_experience/static/course_experience/js/spec/LatestUpdate_spec.js

```diff
@@ -0,0 +1,30 @@
+/* globals $, loadFixtures */
+
+import 'jquery.cookie';
+import { LatestUpdate } from '../LatestUpdate';
+
+
+describe('LatestUpdate tests', () => {
+  function createLatestUpdate() {
+    new LatestUpdate({ messageContainer: '.update-message', dismissButton: '.dismiss-message button' }); // eslint-disable-line no-new
+  }
+  describe('Test cookies', () => {
+    beforeEach(() => {
+      $.cookie('update-message', '');
+      loadFixtures('course_experience/fixtures/latest-update-fragment.html');
+    });
+
+    it('Test dismiss button', () => {
+      createLatestUpdate();
+      expect($('.update-message').attr('style')).toBe(undefined);
+      $('.dismiss-message button').click();
+      expect($('.update-message').attr('style')).toBe('display: none;');
+    });
+
+    it('Test cookie hides update', () => {
+      $.cookie('update-message', 'hide');
```

Comment:
Also, in order to verify that the behavior works the way we expect when we null out the cookie, we still need to reload the fixtures and recreate the update, which doesn't seem like a very useful test either. I just added a new assertion to the first test to verify that the cookie has been voided instead of adding the new second condition.